src/App.js

```diff
@@ -23,16 +39,61 @@ class App extends Component {
     console.log("customerID is:" + customerId);
   }
 
+  // componentDidMount() {
```

Comment:
why not delete this if there is duplicate componentdidmount  with the exact same code?

Reply:
If you want to delete them, you can remove it. But because I think that we use that code after, so I didn't remove it

Reply:
If you want to remove it, I will push removed result when I push new code